docs/gateway.md

```diff
@@ -78,6 +78,71 @@ Options:
   -V, --version                    Print version information
 ```
 
+#### Gateway Peg-in Process
+This section outlines the steps to add a gateway to a federation and subsequently fund the gateway.
+
+##### Connecting the Gateway to a Federation
+Start by connecting your gateway to the desired federation using the following command:
+```bash
+$ gateway-cli connect-fed fed1xxxxxxx
+```
+##### Requesting a Deposit Address
+Once the gateway is successfully integrated into the federation, you can request a new address to deposit coins:
+```bash
+$ gateway-cli address --federation-id fed1xxxxxxx
+bc1xyz...
+```
+##### Sending Coins to the Gateway
+After obtaining the deposit address, you can send coins to this address. Below is an example using `lncli`, but you can use any compatible method:
+```bash
+$ lncli sendcoins bc1xyz... 50000 --min_confs 0 --sat_per_vbyte 12
+{
+    "txid": "1a6..."
+}
+```
+##### Confirming the Transaction
+Ensure that the transaction receives at least 10 confirmations from the Bitcoin network. Once confirmed, the funds will be available in the gateway. You can check the status and see the updated balance as follows:
```

Comment:
It's not always 10, depends on the federation config. `finality_delay + 1` was the correct value iirc?